themes/bootstrap3/templates/header.phtml

```diff
@@ -48,33 +48,37 @@
           <?php endif; ?>
 
           <?php if (isset($this->layout()->themeOptions) && count($this->layout()->themeOptions) > 1): ?>
-            <li class="theme dropdown">
-              <a href="#" class="dropdown-toggle" data-toggle="dropdown"><?=$this->transEsc("Theme")?> <b class="caret"></b></a>
-              <ul class="dropdown-menu">
-                <?php foreach ($this->layout()->themeOptions as $current): ?>
-                  <li<?=$current['selected'] ? ' class="active"' : ''?>>
-                    <a href="<?=$this->escapeHtmlAttr($this->url()->addQueryParameters(['ui' => $current['name']])) ?>" rel="nofollow">
-                      <?=$this->transEsc($current['desc']) ?>
-                    </a>
-                  </li>
-                <?php endforeach; ?>
-              </ul>
-            </li>
+            <?=$this->component('menu-button', [
+              'toggleLabel' => 'Theme',
+              'menuItems' => array_map(
+                  function ($current) {
+                  $item = $current;
+                  $item['label'] = $current['desc'];
+                  $item['url'] = $this->url()->addQueryParameters(['ui' => $current['name']]);
+
+                  return $item;
+                },
+                  $this->layout()->themeOptions
+              )
+            ]) ?>
           <?php endif; ?>
 
           <?php if (isset($this->layout()->allLangs) && count($this->layout()->allLangs) > 1): ?>
-            <li class="language dropdown">
-              <a href="#" class="dropdown-toggle" data-toggle="dropdown"><?=$this->transEsc("Language")?> <b class="caret"></b></a>
-              <ul class="dropdown-menu">
-                <?php foreach ($this->layout()->allLangs as $langCode => $langName): ?>
-                  <li<?=$this->layout()->userLang == $langCode ? ' class="active"' : ''?>>
-                    <a href="<?=$this->escapeHtmlAttr($this->url()->addQueryParameters(['lng' => $langCode])) ?>" rel="nofollow">
-                      <?=$this->displayLanguageOption($langName) ?>
-                    </a>
-                  </li>
-                <?php endforeach; ?>
-              </ul>
-            </li>
+            <?=$this->component('menu-button', [
+              'wrapperClass' => 'language',
+              'toggleLabel' => 'Language',
+              'menuItems' => array_map(
+                  function ($langCode, $langName) {
+                  return [
```

Comment:
Something weird seems to be going on with the indentation here; is php-cs-fixer messing something up?